packages/mongodb-memory-server-core/src/__tests__/testUtils/globalSetup.ts

```diff
@@ -1,13 +1,17 @@
 import MongoBinary from '../../util/MongoBinary';
+import { getOS, isDebian10OrHigher } from '../../util/getos';
 import resolveConfig, { ResolveConfigVariables } from '../../util/resolveConfig';
 import { assertion, isNullOrUndefined } from '../../util/utils';
 
 export = async function globalSetup(): Promise<void> {
   const defaultVersion = resolveConfig(ResolveConfigVariables.VERSION);
   assertion(!isNullOrUndefined(defaultVersion), new Error('Default version is not defined'));
   const versions = [defaultVersion, '4.0.28', '4.2.23', '4.4.22', '5.0.19', '6.0.6'];
+  const isNewDebian = isDebian10OrHigher(await getOS());
   // Ensure all required versions are downloaded for tests
   for (const version of versions) {
-    await MongoBinary.getPath({ version });
+    if (!isNewDebian || parseFloat(version) >= 4.2) {
```

Comment:
instead of float, i would prefer having it use semver

Reply:
Removed